qbplatform/quote/models.py

```diff
@@ -66,5 +125,13 @@ class BonusPartDifficulty(models.TextChoices):
 
     bonus = models.ForeignKey(Bonus, on_delete=models.CASCADE)
 
-    part_number = models.IntegerField()
+    part_number = models.IntegerField(
```

Comment:
Do we really want to do this? I realize that this is correct for 95% of quizbowl, but it does feel like a somewhat unnecessary restriction and would mean we can't use this for more "esoteric" formats.

Reply:
What formats have bonuses with more than three parts? Like lightning rounds?

Reply:
I'm not sure. There were old packets that had weird bonus structures e.g. "for the stated number of points" and while they don't come up often sometimes people will use them as a joke. I'm just not sure if there's any real benefit to enforcing this at the model level; we can warn people if they have too many or too few parts, but I'm reluctant to actually tell them that we absolutely won't support anything other than 3.

Reply:
Okay, that's reasonable. I'll remove the restriction and update the PR